Tools/LyTestTools/ly_test_tools/o3de/editor_test_utils.py

```diff
@@ -184,6 +226,49 @@ def save_failed_asset_joblogs(workspace: ly_test_tools._internal.managers.worksp
                     logger.warning(f"Error when saving log at path: {full_log_path}\n{e}")
 
 
+def split_batched_editor_log_file(workspace: ly_test_tools._internal.managers.workspace.AbstractWorkspaceManager,
+                                  starting_path: str,
+                                  destination_file: str) -> None:
+    """
+    Splits a batched editor log file into separate log files for each test case in the log
+    :param workspace: The LyTestTools Workspace object
+    :param starting_path: the original path for the logs
+    :param destination_file: the destination path for the logs
+    :return: None
+    """
+    if not destination_file:
+        logger.warning(f'No destination_file path found, got {destination_file} instead.')
```

Comment:
Misleading as this is not verifying the file's path, it is verifying a string is not `None` ...probably should verify the path.

Reply:
this is actually copypasta'd from a previous commit that was already merged into the code: https://github.com/o3de/o3de/commit/6e5819796b62f4b1224f992c6974e3bacd8bd05d 

I can still change it though

Reply:
Please do!